service/controller/clusterapi/v31/resource/tccp/template/template_main_record_sets.go

```diff
@@ -67,6 +67,15 @@ const TemplateMainRecordSets = `
       Name: '{{ $v.EtcdDomain }}.'
       HostedZoneId: !Ref 'HostedZone'
       Type: A
+  IngressWildcardRecordSet:
+    Type: AWS::Route53::RecordSet
+    Properties:
+      Name: '*.{{ $v.ClusterID }}.k8s.{{ $v.BaseDomain }}.'
+      HostedZoneId: !Ref 'HostedZone'
+      TTL: '300'
+      Type: CNAME
+      ResourceRecords:
+        - '*.{{ $v.ClusterID }}.k8s.{{ $v.BaseDomain }}.'
```

Comment:
it is `ingress.<domain>`, not `wildcard`

Reply:
Should it be renamed to `IngressRecordSet`?